src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -153,6 +153,7 @@ public function preprocessEvent(Event $event, string $langcode, array $settings)
             'vlieg' => $this->isVliegEvent($event),
             'uitpas' => $this->isUitpasEvent($event),
             'facilities' => $this->getFacilitiesWithPresentInformation($event),
+            'typeId' => $event->getTermsByDomain('eventtype')[0]->getId(),
```

Comment:
Are you sure that `$event->getTermsByDomain('eventtype')` will always return an array with some elements?
If not, this would break, so maybe a check is needed?